### DISCUSSION TITLE
Dúvida sobre Exemplo de App

### DISCUSSION THREAD

**Discussion by araujovelho** [balta-io/forum#303] [Tira Dúvidas] [answered]
Olá professor,

Assisti toda a aula do App referente ao código fonte e instalei no Visual Studio Code, mas ele apresenta vários avisos de problemas no terminal. Seria algum problema de versão? Estou rodando as versões atualizadas do Flutter e Dart.  O Android Studio roda perfeitamente a versão demo criada por padrão. Muito bom o curso. Abs

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @araujovelho, como vai?

É importante que utilize as mesmas versões dos cursos para obter os mesmos resultados.
Depois você pode atualizar para as novas versões.
